scripts/copy_type_description_generated_sources.bash

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Description:
+# Copy the structs defined in type_description_interfaces to rosidl_runtime_c and rosidl_runtime_cpp
+# so that they can be used in code generation to describe interface types
```

Comment:
```suggestion
# so that they can be used in code generation to describe interface types.
```

Also, I think it is worthwhile to repeat here that this is necessary to break the loop between type_description_interfaces and the generators.